docs/source/assemblyguide.rst

```diff
@@ -0,0 +1,339 @@
+* Es hält das Zählrohr in einer Höhe von 1 m über der Wiese – somit sind unsere Messwerte (bestmöglich) vergleichbar zu denen des amtlichen `ODL-Messnetzes <https://odlinfo.bfs.de/DE/index.html>`_.
+
+Rohre
+-----
+
+.. image:: images/build_13_pipe.jpg
+
+* Rohr für das Elektronikgehäuse: Zunächst wird das Rohr mit einer Säge auf entsprechende Länge abgesägt.
+  Die Muffe (Aufweitung) zeigt nach unten.
+* Rohr für den Erdspieß: Der noch umgesägte Teil des Restes ist das obere Ende des Erdspießes.
+  Unten wird der Erdspieß auf entsprechende Länge abgesägt im 45-Grad-Winkel. 
+  Mit diesem schrägen Anschnitt lässt sich der Erdspieß später mit leichten Drehbewegungen gut in die Erde bohren. 
+  Alternativ kann man diesen Teil des Rohrs kurzhalten (ca. 15cm) und in einem verzinkten Sonnenschirmhalter mit Erdspieß (Ø40mm) festklemmen.
+* Lüftungs-/Kabelloch bohren: Mit einem Reißnagel o.ä. wird das Loch seitlich im Erdspieß vorgestochen. 
+  Anschließend wird es aufgebohrt. Zuletzt wird es mit einem großen Bohrer / Schälbohrer auf etwa 17 mm Durchmesser aufgebohrt. 
+
+.. note:: Verletzungsgefahr: Mit der Klinge/dem Bohrer immer nach außen, d.h. vom Körper weg, schneiden oder Bohren!
+
+Dach und Zwischenboden
+----------------------
+
+.. image:: images/build_14_cap.jpg
+   :align: right
+   :width: 20%
+   :alt: Deckel
+
+Für das Dach kann man einen Rohrabschlussstopfen oben vorsichtig über das Rohr schieben, s. Bild rechts.
+
+.. image:: images/build_14_bottle_bottom.jpg
+   :width: 30%
+   :align: left
+   :alt: Zwischenboden
+
+Als Zwischenboden verwenden wir eine Kruke, also eine Plastikdose, die in Apotheken zum Abfüllen von Salben verwendet wird.
+Der rote Deckel kann als Material für eine kabeldurchführungsverkleinernde Zwischenbodenauflage dienen.
+
+Für den Zwischenboden (s. Bild links) wird lediglich das Gewinde vorsichtig mit der Säge abgesägt. Der „Kragen“ muss bestehen bleiben, da er als Anschlag dient.
+
+In den Zwischenboden wird in der Mitte ein Loch zur Kabeldurchführung mit 10 mm gebohrt. Der Stecker des Kabels muss durch das Loch passen.
+
+.. Auskommentiert, da die Variante mit Deckel viel einfacher ist
+  Bei Verwendung einer Kruke als Dach wird der „Kragen“ unter dem Gewinde mit abgesägt, so dass die Dachkruke oben genau in das Rohr hineinpasst:
+
+  .. image:: images/build_14_bottle_top.jpg
+     :width: 30%
+     :align: left
+     :alt: Deckel zum Einkleben
+
+  Das obere Viertel der Dachkruke wird seitlich mit Silikonklebstoff benetzt und in das obere Teil des Elektronikgehäuses geschoben. Die Fuge wird weiterhin mit Silikonklebstoff abgedichtet. 
+
+  Auch der obere Rand der Dachkruke wird mit Klebstoff benetzt (s. Skizze). Um ein Hereinrutschen zu verhindern, kann man sie von unten mit einem langen Gegenstand fixieren. 
+  Die Silikonfuge kann man glätten, indem man einen Finger mit Spülmittel benetzt und damit das „Dach“ glättet. 
+
+.. Ab hier nicht mehr auskommentiert die Anleitung zur Auflage:
```

Comment:
@WwwBoehriDe muessen wir "auskommentierte" Doku haben?

Oder kannst Du entscheiden, was besser ist und nur diese Alternative behalten wir dann?

Reply:
Diskutiert das doch bitte in einem Issue, ich hätte den Pull Request hier gerne möglichst bald durch.